tests/test_warm_reboot.py

```diff
@@ -962,17 +965,18 @@ def test_routing_WarmRestart(dvs, testlog):
     # Enable ipv6 on docker
     dvs.runcmd("sysctl net.ipv6.conf.all.disable_ipv6=0")
 
-    dvs.runcmd("ip -4 addr add 111.0.0.1/24 dev {}".format(intfs[0]))
-    dvs.runcmd("ip -6 addr add 1110::1/64 dev {}".format(intfs[0]))
-    dvs.runcmd("ip link set {} up".format(intfs[0]))
-
-    dvs.runcmd("ip -4 addr add 122.0.0.1/24 dev {}".format(intfs[1]))
-    dvs.runcmd("ip -6 addr add 1220::1/64 dev {}".format(intfs[1]))
-    dvs.runcmd("ip link set {} up".format(intfs[1]))
-
-    dvs.runcmd("ip -4 addr add 133.0.0.1/24 dev {}".format(intfs[2]))
-    dvs.runcmd("ip -6 addr add 1330::1/64 dev {}".format(intfs[2]))
-    dvs.runcmd("ip link set {} up".format(intfs[2]))
+    # Defining create neighbor entries (4 ipv4 and 4 ip6, two each on each interface) in linux kernel
+    intf_tbl = swsscommon.Table(conf_db, "INTERFACE")
+    fvs = swsscommon.FieldValuePairs([("NULL","NULL")])
+    intf_tbl.set("Ethernet0|111.0.0.1/24", fvs)
```

Comment:
Is it possible to use the same `intfs `list here as well for consistency, since that is used throughout the function?